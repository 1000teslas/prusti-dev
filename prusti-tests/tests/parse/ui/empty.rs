// compile-flags: -Pprint_desugared_specs=true -Pskip_verify=true -Phide_uuids=true

fn main() {}
