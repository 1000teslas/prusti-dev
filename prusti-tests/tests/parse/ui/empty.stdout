// compile-flags: -Zprint-desugared-specs -Zskip-verify -Zhide-uuids

#![feature(stmt_expr_attributes)]
#![feature(register_tool)]
#![register_tool(prusti)]
#[prelude_import]
use std::prelude::v1::*;
#[macro_use]
extern crate std;
fn main() { }
